app/components/TrackCard/index.js

```diff
@@ -53,13 +64,18 @@ const CustomButton = styled.div`
 `;
 
 export function TrackCard({ item, currentPlayingId, isPlaying, onPlay, onPause }) {
-  const { artworkUrl100, trackName, artistName, previewUrl, trackId } = item;
+  const { trackName, artistName, artworkUrl100, previewUrl, trackId } = item;
   return (
-    <CustomCard data-testid="track-card" cover={<Image alt="artwork" src={artworkUrl100} />}>
-      <TextCard data-testid="text-card" title={trackName} description={artistName} />
+    <CustomCard data-testid="track-card">
+      <Link to={`/track/${trackId}`}>
```

Comment:
This would mean only the Top portion or the part with the image is clickable. Ideally I would want the whole card to be clickable and routable.

Reply:
but why would we want to route when we click play button , so when we click on image or text part we will route